packages/zowe-explorer-api/src/logger/IZoweLogger.ts

```diff
@@ -60,7 +60,12 @@ export class IZoweLogger {
      *
      */
     public logImperativeMessage(message: string, severity: MessageSeverityEnum): void {
-        const messageWithExtensionName = `Error in extension ${this.extensionName}: ${message}`;
+        let messageWithExtensionName;
+        if (severity < 3) {
+            messageWithExtensionName = `Info from extension ${this.extensionName}: ${message}`;
+        } else {
+            messageWithExtensionName = `Error in extension ${this.extensionName}: ${message}`;
+        }
```

Comment:
@katelynienaber with the different levels laid out in the MessageSeverityEnum values, should this message more generic?  Labeling a warning as an error can be confusing. Since you have the case below I think we could have a generic message here with the parts that are going to be the same in each message and expand on that in the switch/case.  For example:
```
messageWithExtensionName = `Extension ${this.extensionName}: ${message}`;`
```
and the `this.log...` will automatically include the level of severity in brackets before the message in the log, correct?

Reply:
Imho it is important to list the severity in the error message. So I did the following:

```
    public logImperativeMessage(message: string, severity: MessageSeverityEnum): void {
        let messageWithExtensionName = `message from extension ${this.extensionName}: ${message}`;
        switch (severity) {
            case MessageSeverityEnum.TRACE:
                this.log.trace(`TRACE ${messageWithExtensionName}`);
                break;
            case MessageSeverityEnum.DEBUG:
                this.log.debug(`DEBUG ${messageWithExtensionName}`);
                break;
            case MessageSeverityEnum.INFO:
                this.log.debug(`INFO ${messageWithExtensionName}`);
                break;
            case MessageSeverityEnum.WARN:
                this.log.debug(`WARNING ${messageWithExtensionName}`);
                break;
            case MessageSeverityEnum.ERROR:
                this.log.debug(`ERROR ${messageWithExtensionName}`);
                break;
            case MessageSeverityEnum.FATAL:
                this.log.debug(`FATAL ERROR ${messageWithExtensionName}`);
                break;
        }
    }
```

Please let me know if it works for you!